### ISSUE TITLE
How to initialize different account key for ios?

### ISSUE THREAD

**Issue by briankoey** [taskrabbit/react-native-zendesk-chat#37]
For ios initialization, we can do this only for one key.
`[ZDCChat initializeWithAccountKey:@"accountkey"];`

How I able to initialize from react native?. I manage to get it works as the code shown below only for android, but not ios. I need to based on a props value to determine which account key should apply.
```
if (props === 'A') {
ZendeskChat.init('accountKeyA');
}  else {
ZendeskChat.init('accountKeyB');
}
```

Please help

**Comment by wktq** [comment]
I've wanted to same thing and I've imitated some other exported methods like this.

```
RCT_EXPORT_METHOD(init:(NSDictionary *)options) {
  dispatch_sync(dispatch_get_main_queue(), ^{
    [ZDCChat initializeWithAccountKey:options[@"accessKey"]];
  });
}
```
Add this code to RNZendeskChatModule.m and it worked.

but I don't know this is correct way I'm not iOS developer.

**Comment by fbartho** [comment]
This will be resolved by #42 which I hope to ship this month!

**Comment by fbartho** [comment]
@wktq that looks fine, btw, and gave me some ideas!

**Comment by robertmurray** [comment]
This was addressed by #42  and merged into v0.3.0
